### PR TITLE
Add autofill hints to login screen

### PR DESCRIPTION
**Pull request by chenghsuanho**
## Description 
- This PR causes the username and password fields of the login screen to trigger autofill, improving the UX of password manager users.
- The Column widget gets wrapped with the AutofillGroup widget.

## How to Verify? 
- Prepare an password manager, e.g. KeepassXC2Android or Google Passwords
- Launch the app logged out
- Click on either username or password field triggers the autofill prompt.

## Screenshots/GIF/Test Results (Optional) 

username
password

### CODE REVIEW

File: lib/ui/screen/login_screen.dart [hunk 1]
```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
...
-                          errorStyle: const TextStyle(
-                            height: 0,
-                            fontSize: 0,
+                child: AutofillGroup(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Material(
+                        elevation: 2,
+                        borderRadius: const BorderRadius.all(Radius.circular(32)),
```
Comment (ricky9667) [inline lib/ui/screen/login_screen.dart:95]: Suggest changing to `BorderRadius.circular`.

```suggestion
                        borderRadius: const BorderRadius.circular(32),
```

File: lib/ui/screen/login_screen.dart [hunk 2]
```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
...
-                            Icons.lock,
-                            color: Colors.grey,
-                          ),
-                          border: InputBorder.none,
-                          contentPadding: const EdgeInsets.symmetric(
-                            horizontal: 25,
-                            vertical: 13,
+                      Material(
+                        elevation: 2,
+                        borderRadius: const BorderRadius.all(Radius.circular(32)),
```
Comment (ricky9667) [inline lib/ui/screen/login_screen.dart:140]:
```suggestion
                        borderRadius: const BorderRadius.circular(32),
```

File: lib/ui/screen/login_screen.dart [hunk 3]
```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
...
-                      Padding(
-                        padding: const EdgeInsets.only(left: 16),
-                        child: Text(
-                          _accountErrorMessage,
-                          style: const TextStyle(
-                            fontSize: 12,
-                            color: Colors.red,
+                      const SizedBox(
+                        height: 4,
+                      ),
```
Comment (ricky9667) [inline lib/ui/screen/login_screen.dart:123]: [nit]

Could you help us remove the trailing comma for widgets with single properties?
Although we don't have very strict conventions, it makes code look nicer in my opinion.

```suggestion
                      const SizedBox(height: 4),
```

File: lib/ui/screen/login_screen.dart [hunk 4]
```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
...
+                          controller: _accountControl,
+                          cursorColor: Colors.blue[800],
+                          textInputAction: TextInputAction.next,
+                          focusNode: _accountFocus,
+                          autofillHints: const [AutofillHints.username],
+                          onEditingComplete: () {
+                            _accountFocus.unfocus();
+                            FocusScope.of(context).requestFocus(_passwordFocus);
+                          },
+                          validator: (value) => _validatorAccount(value),
```
Comment (Xanonymous-GitHub) [inline lib/ui/screen/login_screen.dart:106]:
```suggestion
                          validator: _validatorAccount,
```

[Help improve]
Suggest directly passing function itself, without wrapping a anonymous function.

File: lib/ui/screen/login_screen.dart [hunk 5]
```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
...
+                          controller: _passwordControl,
+                          cursorColor: Colors.blue[800],
+                          obscureText: true,
+                          focusNode: _passwordFocus,
+                          autofillHints: const [AutofillHints.password],
+                          onEditingComplete: () {
+                            _passwordFocus.unfocus();
+                            TextInput.finishAutofillContext();
+                          },
+                          validator: (value) => _validatorPassword(value),
```
Comment (Xanonymous-GitHub) [inline lib/ui/screen/login_screen.dart:151]:
```suggestion
                          validator: _validatorPassword,
```

ditto.

File: lib/ui/screen/login_screen.dart (hunk 2 above)
Comment (chenghsuanho) [reply lib/ui/screen/login_screen.dart:140]: Sure, done in [5539c2b](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/5539c2b6d8b336a021e5f81cabe5d4323107fffb).

Comment (chenghsuanho) [reply lib/ui/screen/login_screen.dart]: Done in [9d92f57](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/9d92f57c53a7bda5eebc22af86d0d46c1833ba45).

File: lib/ui/screen/login_screen.dart (hunk 3 above)
Comment (chenghsuanho) [reply lib/ui/screen/login_screen.dart]: Done in [9d92f57](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/9d92f57c53a7bda5eebc22af86d0d46c1833ba45).

File: lib/ui/screen/login_screen.dart (hunk 4 above)
Comment (chenghsuanho) [reply lib/ui/screen/login_screen.dart]: Done in [9d92f57](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/9d92f57c53a7bda5eebc22af86d0d46c1833ba45).

File: lib/ui/screen/login_screen.dart (hunk 5 above)
Comment (chenghsuanho) [reply lib/ui/screen/login_screen.dart]: Done in [9d92f57](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/9d92f57c53a7bda5eebc22af86d0d46c1833ba45).

**Review by Xanonymous-GitHub — APPROVED** [review]
LGTM.

**Review by ricky9667 — APPROVED** [review]
LGTM.

**Pull request merged**